### DISCUSSION TITLE
Crash when pressing [Enter] when using "Generating code from a comment" feature

### DISCUSSION THREAD

**Discussion by ghost** [community/community#13983] [Copilot Conversations]
Hello, 

On visual studio 2022 I get a complete visual studio crash when using "Generating code from a comment" feature. Step to get this crash :  

- Adding new comment beginning by "//"
- Press "Enter", suggestion appear
- Press "Enter", visual studio crash happen

Language : C#
OS : Windows 11 Professional version 21H build 22000.556
Visual Studio Version : Microsoft Visual Studio Community 2022 Version 17.1.0
GitHub copilot version : 1.10.0.2

**Comment by kzu** [comment]
This is fixed in v1.12+
